Lang/Spanish.pj.Lang

```diff
@@ -419,7 +420,7 @@
 //Edit Cheat
 #1027#  "Editar Truco"
 #1028#  "Actualizar Truco"
-#1029#  "El Truco ha sido cambiado.\n\n¿Deseas actualizar?"
+#1029#  "El Truco ha sido cambiado.\n\n¿Quieres actualizar?"

```

Comment:
I'm talking about formal/business writing.
Can you name a piece of professional, even commercial software that offers options using "Quieres" instead of "deseas"?

I recall providing you an example of some Microsoft website, translated to Spanish, that uses the latter.

Again, please don't use translators for semantics or redaction accuracy. If those tools were an appropriate source, there would be no people translating books or movies.
Such tools are great for basic translations, but fall short for more complex or subtle texts.